### PR TITLE
Fixed EN toggle metric publishing. It now only gets published when th…

### PR DESCRIPTION
**Pull request by craigzour**
# Summary | Résumé

- Fixed EN toggle metric publishing. It now only gets published when the user manually turn it on or off.

### CODE REVIEW

File: src/screens/home/views/ExposureNotificationsDisabledView.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const ExposureNotificationsDisabledView = () => {
   }, []);
 
   const startEn = useCallback(async () => {
-    await startExposureNotificationService();
+    await startExposureNotificationService(true);
```
Comment (smcmurtry) [inline src/screens/home/views/ExposureNotificationsDisabledView.tsx:19]: change to `false`

File: src/screens/home/views/ExposureNotificationsUnauthorizedView.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ export const ExposureNotificationsUnauthorizedView = () => {
   }, []);
 
   const startEn = useCallback(() => {
-    startExposureNotificationService();
+    startExposureNotificationService(true);
```
Comment (smcmurtry) [inline src/screens/home/views/ExposureNotificationsUnauthorizedView.tsx:19]: change to `false`

File: src/screens/home/views/LocationOffView.tsx [hunk 3]
```diff
@@ -16,7 +16,7 @@ export const LocationOffView = () => {
   }, []);
 
   const startEn = useCallback(() => {
-    startExposureNotificationService();
+    startExposureNotificationService(true);
```
Comment (smcmurtry) [inline src/screens/home/views/LocationOffView.tsx:19]: change to `false`

File: src/screens/menu/views/SystemStatusUnauthorized.tsx [hunk 4]
```diff
@@ -9,7 +9,7 @@ export const SystemStatusUnauthorized = () => {
   const startExposureNotificationService = useStartExposureNotificationService();
   const onPress = async () => {
     if (Platform.OS === 'android') {
-      await startExposureNotificationService();
+      await startExposureNotificationService(true);
```
Comment (smcmurtry) [inline src/screens/menu/views/SystemStatusUnauthorized.tsx:12]: change to `false`

File: src/screens/menu/views/SystemStatusOff.tsx [hunk 5]
```diff
@@ -9,7 +9,7 @@ export const SystemStatusOff = () => {
   const startExposureNotificationService = useStartExposureNotificationService();
   const onPress = async () => {
     if (Platform.OS === 'android') {
-      await startExposureNotificationService();
+      await startExposureNotificationService(true);
```
Comment (smcmurtry) [inline src/screens/menu/views/SystemStatusOff.tsx:12]: change to `false`

**Review by smcmurtry — CHANGES_REQUESTED** [review]
Since this metric is to figure out how much people are using the on/off feature, it should just be sent from these 2 places:
- the on/off button in the menu
- the "Turn back on" button on the `ExposureNotificationsUserStoppedView`

Let's remove it from the "on" buttons on the other error screens, since those are not specific to the on/off feature.

File: src/screens/menu/views/TurnAppBackOn.tsx [hunk 6]
```diff
@@ -8,7 +8,7 @@ export const TurnAppBackOn = () => {
   const startExposureNotificationService = useStartExposureNotificationService();
 
   const onStart = useCallback(async () => {
-    await startExposureNotificationService();
+    await startExposureNotificationService(true);
```
Comment (craigzour) [inline src/screens/menu/views/TurnAppBackOn.tsx:11]: @smcmurtry You did not leave any comment here. Should I also set it to `false` ?

Comment (smcmurtry) [reply src/screens/menu/views/TurnAppBackOn.tsx:11]: I just checked and it looks like the `TurnAppBackOn` component isn't used anywhere. So it doesn't matter - this whole file could be deleted.

Comment (feralbob) [reply src/screens/menu/views/TurnAppBackOn.tsx:11]: s/could/should

😀

Comment (craigzour) [reply src/screens/menu/views/TurnAppBackOn.tsx]: Haha! I was in the process of doing it :)

**Pull request merged**